src/dynarec/la64/dynarec_la64_f30f.c

```diff
@@ -331,6 +331,51 @@ uintptr_t dynarec64_F30F(dynarec_la64_t* dyn, uintptr_t addr, uintptr_t ip, int
                 SMWRITE2();
             }
             break;
+        case 0xB8:
+            INST_NAME("POPCNT Gd, Ed");
+            SETFLAGS(X_ALL, SF_SET);
+            SET_DFNONE();
+            nextop = F8;
+            v1 = fpu_get_scratch(dyn);
+            GETED(0);
+            GETGD;
+            if (MODREG) {
+                if (rex.w) {
+                    VINSGR2VR_D(v1, ed, 0);
+                    VINSGR2VR_D(v1, xZR, 1);
+                } else {
+                    ZEROUP2(x4, ed); // need to clear uper part
+                    ed = x4;
+                    VINSGR2VR_D(v1, ed, 0);
+                    VINSGR2VR_D(v1, xZR, 1);
+                }
```

Comment:
Clear v1 first with `VXOR_V` first, then you can use `VINSGR2VR_W/D` to do the job.